plugins/tours/src/admin/wp-list-table.php

```diff
@@ -58,14 +58,15 @@ function _render_custom_column_content( $column_name, $tour_id ) {
 /**
  * Set sortable columns on the Tours admin page.
  *
- * @return array New sortable columns.
+ * @param array $sortable_columns An array of sortable columns.
+ * @return array  New sortable columns.
  * @since 1.0.0
  *
  */
-function _set_sortable_columns() {
-	return array(
+function _set_sortable_columns( $sortable_columns ) {
+	return [
 		'tour_id'    => 'Tour ID',
 		'tour_year'  => 'Tour Year',
 		'menu_order' => 'Order Number',
-	);
+	];
 }
```

Comment:
Make sure you have an empty line at the end of each PHP file.

Reply:
@hellofromtonya Fixed.